src/app/index.tsx

```diff
@@ -37,4 +39,7 @@ const App = () => {
     </AppContext.Provider>
   )
 }
-export default App
+
+export default withErrorBoundary(App, {
+  FallbackComponent: () => <ErrorPage />,
```

Comment:
```suggestion
  FallbackComponent: ErrorPage,
```
по доке вижу, что можно сразу так